src/esp/sensor/PinholeCamera.cpp

```diff
@@ -110,6 +110,10 @@ void PinholeCamera::drawObservation(sim::Simulator& sim) {
     // TODO: check sim has semantic scene graph
     renderer->draw(*this, sim.getActiveSemanticSceneGraph(),
                    sim.isFrustumCullingEnabled());
+    if (&sim.getActiveSemanticSceneGraph() != &sim.getActiveSceneGraph()) {
```

Comment:
I probably just missed something in the previous PRs -- there are two independent scenegraphs now? :)

Reply:
Unfortunately yes in some cases only which this PR is addressing. When we load MP3D for example we load an RGB-D asset in to one SceneGraph (where objects also go) and a separate semantic asset into another SceneGraph. This PR enables SEMANTIC sensor to render only the objects in the RGB-D SceneGraph as a 2nd pass after rendering the semantic SceneGraph.

Reply:
@mosra: This (two scene graphs) was not newly introduced to the system. They were here in the system since day 1, when the semantic mesh was added.